mathics/builtin/assignment.py

```diff
@@ -625,6 +627,11 @@ class Definition(Builtin):
     >> Definition[a]
      = a = 2
 
+    >> a = 2;
+    >> ? a
+     = a = 2
+
+
```

Comment:
I think the point in the docs is (it's unclear) that under the hood `Information` is calling `Definitions` to construct its output.